ui-tests/jupyter_server_test_config.py

```diff
@@ -4,17 +4,9 @@
 opens the server to the world and provide access to JupyterLab
 JavaScript objects through the global window variable.
 """
-from tempfile import mkdtemp
+from jupyterlab.galata import configure_jupyter_server
 
-c.ServerApp.port = 8888  # noqa: F821
-c.ServerApp.port_retries = 0  # noqa: F821
-c.ServerApp.open_browser = False  # noqa: F821
-
-c.ServerApp.root_dir = mkdtemp(prefix="galata-test-")  # noqa: F821
-c.ServerApp.token = ""  # noqa: F821
-c.ServerApp.password = ""  # noqa: F821
-c.ServerApp.disable_check_xsrf = True  # noqa: F821
-c.LabApp.expose_app_in_browser = True  # noqa: F821
+configure_jupyter_server(c)
```

Comment:
is this a new way to configure the server?

we need to add `# noqa: F821` at the end of this line so the lint doesn't complain

Reply:
Yes, galata provides the default configuration.

Reply:
I added `# noqa: F821`, thanks for the hint